shell/platform/linux/fl_clipping_view.h

```diff
@@ -0,0 +1,51 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#ifndef FLUTTER_SHELL_PLATFORM_LINUX_FL_CLIPPING_VIEW_H_
+#define FLUTTER_SHELL_PLATFORM_LINUX_FL_CLIPPING_VIEW_H_
+
+#include <gtk/gtk.h>
+
+#include <glib-object.h>
+
+G_BEGIN_DECLS
+
+G_DECLARE_FINAL_TYPE(FlClippingView,
+                     fl_clipping_view,
+                     FL,
+                     CLIPPING_VIEW,
+                     GtkEventBox)
+
+/**
+ * FlClippingView:
+ *
+ * #FlClippingView is a GTK widget that draws its child with mutators.
```

Comment:
Does this view clip the child view? Might make sense to add that here.

Reply:
Done.